k8s/dev/supply-chain-worker.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
```

Comment:
@tiagojsag This is the deployment I pushed to have a background worker, but I don't have any insight into why it's not working. Could you take a look or pass any logs over to me over email when you have a moment? I suspect it's RBAC related on the REDIS_URL or the deployment itself. Thanks.

Reply:
sure, please email me,so I can reply to you with the logs.